src/ledger/LedgerManagerImpl.cpp

```diff
@@ -1560,6 +1560,16 @@ LedgerManagerImpl::applyTransactions(
             storeTransaction(mApp.getDatabase(), ledgerSeq, tx, tm.getXDR(),
                              txResultSet);
         }
+        else if (mApp.getConfig().MODE_STORES_HISTORY_MINIMAL)
+        {
+            auto ledgerSeq = ltx.loadHeader().current().ledgerSeq;
+            // Clear the meta as we don't want to store it.
+            tm.clearOperationMetas();
+            tm.clearTxChangesAfter();
+            releaseAssert(xdr::xdr_size(tm.getXDR()) == 0);
+            storeTransaction(mApp.getDatabase(), ledgerSeq, tx, tm.getXDR(),
+                             txResultSet);
+        }
```

Comment:
Looks like the change is incomplete: we're still recording meta in `storeTransactionFee` and `Upgrades::storeUpgradeHistory`

Reply:
I think `storeUpgradeHistory` and `storeTransactionFee` are only invoked when `MODE_STORES_HISTORY_MISC` is true?

Reply:
`MODE_STORES_HISTORY_MISC` it always true on validators, so we're still storing unneeded meta at the moment